optuna/samplers/_tpe/sampler.py

```diff
@@ -948,9 +551,10 @@ def _get_multivariate_observation_pairs(
     scores = []
     values: Dict[str, List[Optional[float]]] = {param_name: [] for param_name in param_names}
     for trial in study.get_trials(deepcopy=False, states=states):
-        # If ``group`` = True, there may be trials that are not included in each subspace.
-        # Such trials should be ignored here.
-        if any([param_name not in trial.params for param_name in param_names]):
+        # If ``multivariate`` = True and ``group`` = True, we ignore the trials that are not
+        # included in each subspace.
+        # If ``multivariate`` = False, we consider such trials.
+        if multivariate and any([param_name not in trial.params for param_name in param_names]):
```

Comment:
In my understanding, the procedure of calculation does not depend of `multivariate`, because `multivariate` is always true when `any([param_name not in trial.params for param_name in param_names])` is true. If my understanding is correct, I think we can omit `multivariate` from the arguments of this method.

Reply:
Sorry for the confusion. Here, we want to check `any([param_name not in trial.params for param_name in param_names])` only when `multivariate = True`. In the current master, we don't check `any([param_name not in trial.params for param_name in param_names])` when `multivariate = False`, so for the consistency, I include `multivariate` here.

Reply:
I updated the code comment like: `If multivariate = False, we skip the check`